projects/plugins/jetpack/modules/masterbar/inline-help/inline-help-template.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    // phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+
+	// See https://github.com/Automattic/jetpack/blob/41fe374f32d0fa35bb66b4ca492e0b337ee7c6cd/projects/packages/tracking/src/js/tracks-ajax.js#L29
+	$jp_tracking_classname = 'jptracks';
+?>
+
+<div class="a8c-faux-inline-help">
+	<a data-jptracks-name="<?php echo esc_attr( $args['tracking_event_name'] ); ?>" class="<?php echo esc_attr( $jp_tracking_classname ); ?> a8c-faux-inline-help__button" href="<?php echo esc_url( $args['href'] ); ?>" target="_blank" rel="noopener noreferrer" title="<?php echo esc_attr__( 'Help', 'jetpack' ); ?>">
```

Comment:
Not use if the tracks setup is conflicting with the link, but clicking on the button does not take me to wordpress.com/help, although I can see the `href` attribute properly set:

![Feb-23-2021 14-41-36](https://user-images.githubusercontent.com/1233880/108851923-4590a500-75e5-11eb-82bc-c5307402658a.gif)